content/bn/releases/patch-releases.md

```diff
@@ -0,0 +1,94 @@
+---
+title: প্যাচ রিলিজ
+type: docs
+---
+
+কুবারনেটিস প্যাচ রিলিজের সময়সূচি এবং দলের যোগাযোগ তথ্য।
+
+কুবারনেটিস রিলিজ সাইকেলের সাধারণ তথ্যের জন্য, 
+[রিলিজ প্রক্রিয়া বর্ণনা][release process description] দেখুন।
+
+## ক্যাডেন্স(Cadence)
+
+আমাদের সাধারণ প্যাচ রিলিজ ক্যাডেন্সের মাসিক। এটা
+সাধারণত একটু দ্রুত (1 থেকে 2 সপ্তাহ) হলেও, যখন একটি 1.X মাইনর 
+রিলিজের পরে প্যাচ রিলিজের প্রথমটি হয়। গুরুত্বপূর্ণ বাগ সংশোধন আরও
+সাধারণ ক্যাডেন্সের বাইরে একটি আগামী রিলিজ সৃষ্টি করতে পারে। আমরা 
+এছাড়াও লক্ষ্য করি যে প্রধান ছুটির সময়ে রিলিজ করা হবে না।
+
+## যোগাযোগ
+
+প্যাচ রিলিজ দলের সম্পূর্ণ যোগাযোগের বিস্তারিত তথ্যের জন্য [রিলিজ ম্যানেজার পৃষ্ঠা][release-managers] দেখুন।
+
+দয়া করে আমাদেরকে একটি কার্য দিন দিন - আমরা ভিন্ন টাইমজোন অনুযায়ী থাকতে পারি!
+
+রিলিজের মধ্যে দলটি প্রতি সপ্তাহের ভিতরে আসা চেরি পিক অনুরোধগুলি দেখছে।
+দলটি চেরি পিক অনুরোধকারীদের সাথে GitHub PR, SIG চ্যানেল (স্ল্যাকে)
+এবং স্ল্যাকের  [email](mailto:release-managers-private@kubernetes.io)
+মাধ্যমে যোগাযোগ করবে, এবং যদি পিআরে কোনো
+প্রশ্ন থাকে।
+
+## চেরি পিক
+
+[চেরি পিক প্রসেস][cherry-picks] অনুসরণ করুন।
+
+চেরি পিক গুলির জন্য গিটহাবে পার্শ্ববর্তী লেবেলসহ (উদাহরণস্বরূপ
+`approved`, `lgtm`, `release-note`) এবং চেরি পিকের শেষকার পূর্বে CI টেস্ট পাস করতে হবে।
+এটা সাধারণত লক্ষ্য করা হয় লক্ষ্য রিলিজের দুই দিন পূর্বে, তবে এটা আরও হতে পারে। 
+পিআরের প্রস্তুতি যে পরিপ্রেক্ষিতে তা অনেক ভাল, কারণ আমাদের যখন 
+চেরি পিক আপনার চেরি পিক মানচিত্রে মারার পূর্বে CI সিগনাল পেতে 
+সময় লাগে।
+
+চেরি পিক PR গুলো, যেগুলো মার্জ মানদন্ড মিস করবে, সেগুলো অনুসরণ এবং ট্রাক করা হবে
+পরবর্তী প্যাঁচ রিলিজ এর জন্য ।
+
+## সাপোর্ট পিরিয়ড
+
+[বার্ষিক সাপোর্ট KEP][yearly-support] অনুসারে, কুবারনেটিস কমিউনিটি
+প্রায় চৌদ্দ (১৪) মাসের জন্য সক্রিয় প্যাচ রিলিজ সিরিজের
+সমর্থন করবে।
+
+এই সময়সীমার প্রথম বারো মাসগুলি স্ট্যান্ডার্ড পিরিয়ড হিসাবে
+গণ্য হবে।
+
+বারো মাসের শেষে, নিম্নলিখিত ঘটনা ঘটবে:
+
+- [রিলিজ ম্যানেজার][release-managers] একটি রিলিজ কাটবে
+- প্যাচ রিলিজ সিরিজটি মেইন্টেনেন্স মোডে প্রবেশ করবে
+
+দুই মাসের মেইন্টেনেন্স মোডে অবস্থানের সময়সীমার দায়িত্ব মোডে রিলিজ
+ম্যানেজাররা অতিরিক্ত মেইন্টেনেন্স রিলিজ কাটতে পারেন যাতে নিম্নলিখিত সমস্যাগুলি সমাধান করা যায়:
+
+- CVEs (সিকিউরিটি সংজ্ঞায়িত পরিষদের পরামর্শে)
+- ডিপেন্ডেন্সি সমস্যাগুলি (বেস ইমেজ আপডেট সহ)
+- গুরুত্বপূর্ণ কোর কম্পোনেন্ট সমস্যাগুলি
+
+দুই মাসের মেইন্টেনেন্স মোড সময়সীমার শেষে, প্যাচ রিলিজ সিরিজটি EOL (end of life)
+হিসাবে গণ্য হবে এবং সম্পর্কিত ব্রাঞ্চে চেরি পিক 
+শীঘ্রই পরে বন্ধ করা হবে 
+
+মনে রাখবেন যে, রক্ষণাবেক্ষণ মোড এবং EOL লক্ষ্যের জন্য মাসের 28 তারিখ বেছে নেওয়া হয়েছিল
+সরলতার জন্য (প্রতি মাসে এটি আছে)।
+
+## আগামী মাসিক রিলিজ
+
+বাগ ফিক্সের গুরুত্বের সাথে সময়সীমা পরিবর্তন করতে পারে, তবে আমরা সহজে পরিকল্পনা করতে
+নিম্নলিখিত মাসিক রিলিজ পয়েন্টগুলির লক্ষ্য করব। অপরিকল্পিত, 
+গুরুত্বপূর্ণ রিলিজগুলি এদের মধ্যেও ঘটতে পারে।
+
+{{< upcoming-releases >}}
```

Comment:
Based on the current failed build logs [here](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/6637791355f44f00084d1f48)

> 1:19:31 PM: Error: error building site: assemble: "/opt/build/repo/content/bn/releases/patch-releases.md:79:1": failed to extract shortcode: template for shortcode "upcoming-releases" not found

It looks like this `upcoming-releases` shortcode's html file is missing for the branch being built (dev-1.24-bn.1). To resolve this issue, I suggest adding the [upcoming-releases.html (source here)](https://github.com/kubernetes/website/blob/main/layouts/shortcodes/upcoming-releases.html) file from the **main** branch to this pull request branch [dev-1.24-bn.1 (location here)](https://github.com/kubernetes/website/tree/dev-1.24-bn.1/layouts/shortcodes) . Once you do that, the current build error should get resolved.